deploy-automation.sh

```
@@ -0,0 +1,106 @@
+echo "📦 Staging automation files..."
+
+# Core automation services
+git add lib/services/contract-eligibility.ts
+git add lib/services/contract-rate-calculator.ts
+git add lib/services/transaction-generator.ts
+
+# API endpoints
+git add app/api/automation/
+
+# Settings page
+git add app/(admin)/settings/automation/
```

Shellcheck (0.11.0)

[error] '(' is invalid here. Did you forget to escape it?

(SC1036)